config/globalSetup.ts

```diff
@@ -0,0 +1,4 @@
+export default async () => {
+  process.env.TZ = 'UTC';
```

Comment:
Just wondering, why is this required?

Reply:
Also, in our `.editorconfig` we have `insert_final_newline = true` but extra trailing whitespace shouldn't be allowed.

If you're using VS Code, I'd suggest using the following extensions:
```
christian-kohler.npm-intellisense
EditorConfig.EditorConfig
esbenp.prettier-vscode
```

Paired with `.vscode/settings.json` this should auto-format common source file types on save.

Reply:
Yeah, I am all down for prettier!

Reply:
I've also noticed that `.ts` files are not properly linted (no vs code error), it just throws an error when running `yarn lint`

Reply:
Interesting 🤔 

My IDE (WebStorm) shows a linter error when I removed the ignore in the logger file:
![image](https://user-images.githubusercontent.com/8126518/154537206-aadc841b-43c2-43bd-8836-fd685d89a618.png)

It's only a warning in our linter... so that's why it's highlighted in this yellow shade. Otherwise it would have a red underline for an error. 

My guess is it is a configuration for VSCode? Maybe @vojtechszocs can share his configuration?

Reply:
I think the issue here is we generally run ESLint on each package's sources, and this PR introduces a source directory outside the typical monorepo structure.

I'm actually in favor of moving Jest infra related code into a dedicated monorepo package, but it can be done as a second step.